src/commands/Miscellaneous/discordstatus.ts

```diff
@@ -15,27 +15,121 @@ export default class extends SteveCommand {
 
 	public async run(msg: KlasaMessage): Promise<Message> {
 		const url = 'https://srhpyqt94yxb.statuspage.io/api/v2/summary.json';
-		const embedData = msg.language.tget('commandDiscordStautsEmbed');
-
-		return fetch(url, { method: 'Get' })
-			.then(res => res.json())
-			.then(json => {
-				const embed = new MessageEmbed()
-					.setTitle(json.status.description)
-					.setDescription(embedData.decription(json.status.indicator))
-					.addFields(json.components.map((component: { name: any; status: any }) => ({
+		const embedData = msg.language.tget('commandDiscordStatusEmbed');
+
+		const currentStatus = await fetch(url, { method: 'Get' }).then(r => r.json()) as DiscordStatus;
```

Comment:
axios should probably be used here over fetch for consistency with the rest of the bot, but it's pretty simple. the xkcd and spacepic commands have examples of how to use it

Reply:
Resolved by [94ef5d6](https://github.com/tuataria/steve/pull/63/commits/94ef5d6eb09b30eb62ab5c94c74e399f9657f907)